docs/en/1-Experiments/Jupyter.md

```diff
@@ -65,26 +65,36 @@ available.**
 
 <!-- prettier-ignore -->
 ??? tip "Make sure to restart the Jupyter kernel after installing new software"
-    If you install software in a terminal, but your Jupyter kernel was already
-    running, then it will not be updated.
+    If you install software in a terminal, but your Jupyter kernel was already running, then it will not be updated.
 
 <!-- prettier-ignore -->
 ??? tip "Is there something that you can't install?"
-    If you need something installed, reach us or
-    [open a GitHub issue](https://github.com/StatCan/aaw-kubeflow-containers). We
-    can add it to the default software.
+    If you need something installed, reach us or [open a GitHub issue](https://github.com/StatCan/aaw-kubeflow-containers). We can add it to the default software.
 
 # Once you've got the basics ...
 
 ## Getting Data in and out of Jupyter
 
-You can upload and download data to/from JupyterHub directly in the menu. There
-is an upload button at the top, and you can right-click most files or folders to
-download them.
+You can upload and download data to/from JupyterHub directly in the menu. There is an upload button at the top, and you can right-click most files or folders to download them.
 
 ### Shareable "Bucket" storage
 
-There is also a mounted `minio` folder in your home directory, which holds files
-in [MinIO](../Storage.md/#buckets-via-minio).
+There is also a mounted `minio` folder in your home directory, which holds files in [MinIO](../Storage.md/#buckets-via-minio).
```

Comment:
This will be outdated soon once we deprecate minio. The mounted folder is called `buckets`

Reply:
Thanks! I made the change from `minio` to `buckets`.

Reply:
Strange how it says I dismissed your review. I accept your review.

![image](https://user-images.githubusercontent.com/8212170/228850876-7cccaf6d-1a2b-439e-93b8-1ba486e16a83.png)